pkg/kube/resource_policy.go

```diff
@@ -24,3 +24,23 @@ const ResourcePolicyAnno = "helm.sh/resource-policy"
 // This resource policy type allows resources to skip being deleted
 //   during an uninstallRelease action.
 const KeepPolicy = "keep"
```

Comment:
I was on the fence about this, should we just remove `KeepPolicy` since we never actually need to reference it, or at least make these policy consts unexported?